drivers/sensor/am2320/am2320.c

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) https://github.com/tidyjiang8
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <i2c.h>
+#include <init.h>
+#include <misc/__assert.h>
+#include <misc/byteorder.h>
+#include <sensor.h>
+#include <string.h>
+#include <crc16.h>
+#include <logging/sys_log.h>
+
+/* am2320 i2c address, 7+1 bit */
+#define AM2320_I2C_ADDR     0xB8
```

Comment:
Keep this define name for the 7bit address and use a different define for 7+1bit. This way you don't need a shift for each i2c_write.